src/components/info/templates/lignes-bus-template.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import { defineProps } from 'vue'
+defineProps({
+  layers: {
+    type: Object,
```

Comment:
I am expecting an array of layers here, or am i missing something? Can we have a better typing than `Object`?

Reply:
Right, this template has actually not been migrated yet, but was just added to test the templating logic. I've added the `FeatureInfoJSON` type.